libnetdata/socket/socket.c

```diff
@@ -300,8 +302,18 @@ void listen_sockets_close(LISTEN_SOCKETS *sockets) {
     sockets->failed = 0;
 }
 
+WEB_CLIENT_ACL read_acl(char *st) {
+    if (!strcmp(st,"dashboard")) return WEB_CLIENT_ACL_DASHBOARD;
+    if (!strcmp(st,"registry")) return WEB_CLIENT_ACL_REGISTRY;
+    if (!strcmp(st,"badges")) return WEB_CLIENT_ACL_BADGE;
+    if (!strcmp(st,"management")) return WEB_CLIENT_ACL_MGMT;
+    if (!strcmp(st,"streaming")) return WEB_CLIENT_ACL_STREAMING;
+    if (!strcmp(st,"netdata.conf")) return WEB_CLIENT_ACL_NETDATACONF;
+    return WEB_CLIENT_ACL_NONE;
+}
```

Comment:
I think `all` is also needed, otherwise people will need to pipe all of them.
I think `user` is also needed to enable `dashboard`, `registry` and `badges`.

Reply:
Check this:
I think `all` is also needed, otherwise people will need to pipe all of them.
I think `user` is also needed to enable dashboard, registry and badges.